pip-dep/requirements.txt

```diff
@@ -10,7 +10,7 @@ Pillow~=6.2.2
 requests~=2.22.0
 RestrictedPython~=5.0
 scipy~=1.4.1
-syft-proto~=0.3.0
+syft-proto~=0.4.1
```

Comment:
Will change this back since the current PR does not have some changes required for tests to be able to pass -- will bump the version in a future PR